neon_mq_connector/connector.py

```diff
@@ -92,10 +93,37 @@ def mq_credentials(self):
                                      self.config['MQ']['users'][self.service_name].get('password', 'guest'))
 
     @staticmethod
-    def create_service_id():
-        """Method for generating unique service id, must be invoked on every instance instantiation"""
+    def create_unique_id():
+        """Method for generating unique id"""
```

Comment:
Change this to read "message id" or "request id"; something to specify that this isn't a socket/client ID, but a message/request ID.

Reply:
Discussed in meeting; this will be moved to neon-utils for generic application